test/gmail-adapter.spec.js

```diff
@@ -4,25 +4,34 @@ var Juttle = require('juttle/lib/runtime').Juttle;
 var GmailAdapter = require('../');
 var check_juttle = juttle_test_utils.check_juttle;
 var expect = require('chai').expect;
+var read_config = require('juttle/lib/config/read-config');
```

Comment:
Not sure--from looking at the other adapters that have tests (elastic, mysql, influx), they all have juttle as a peerDependency and install juttle by hand. I think I'd like to keep that for consistency.